workflows/requests.go

```diff
@@ -1,20 +1,36 @@
 package workflows
 
-type WorkflowReference struct {
-	InstanceID string `json:"instance_id"`
+// StartRequest is the struct describing a start workflow request.
+type StartRequest struct {
```

Comment:
Just for my understanding, are these structs used in the runtime too?

Reply:
Yes, in the runtime's universal API handlers, we create instances of these structs and pass them to the component implementations.